### PR TITLE
Sort

### PR DESCRIPTION
**Pull request by kevinzaft**
sum1 test this

### CODE REVIEW

**Comment by ktw1016** [comment]
nice commit msgs

File: BankABC.cpp [hunk 1]
```diff
@@ -204,15 +204,24 @@ inline void Transaction::setAmount(double amountTr)
...
-
+	int i = 0;
+	for (i = 0; i <= K_SizeMax - 1; i++) {
+		for (int j = i + 1; j <= K_SizeMax; j++) {
+			if (*(list + j) == NULL) {
+				break;
+			}
+			int accId1 = (*(list + i))->getAccountId();
+			int accId2 = (*(list + j))->getAccountId();
+			accId1 == 0 ? accId1 = 999999;
```
Comment (Digized) [inline BankABC.cpp:215]: whats happening here?

Comment (ktw1016) [reply BankABC.cpp:215]: yea kevin wut

Comment (kevinzaft) [reply BankABC.cpp:215]: 0 needs to be at the end so i check if the accountID is 0 then "make" it look like the largest number

Comment (ktw1016) [reply BankABC.cpp:215]: noob

Comment (Digized) [reply BankABC.cpp:215]: dont you need a `:` with that operator?

Comment (kevinzaft) [reply BankABC.cpp:215]: Why. You don't need a else in this case since the variable already the value.

Comment (ktw1016) [reply BankABC.cpp:215]: the variable already the value

Comment (kevinzaft) [reply BankABC.cpp:215]: I like how you never have anything useful to contribute :)

Comment (ktw1016) [reply BankABC.cpp:215]: cuz ur code has nothing useful to contribute

Comment (StephenA1) [reply BankABC.cpp:215]: what's wrong with his code?

Comment (Digized) [reply BankABC.cpp:215]: It was giving me errors so I thought maybe its broken

**Pull request merged**